src/geometry/LineUtil.js

```diff
@@ -240,3 +241,59 @@ export function _flat(latlngs) {
 	console.warn('Deprecated use of _flat, please use L.LineUtil.isFlat instead.');
 	return isFlat(latlngs);
 }
+
+/* @function polylineCenter(latlngs: LatLng[], map: Map): LatLng
+ * Returns the center of the `Polyline` of the passed LatLngs.
+ * The returned LatLng will be on the `Polyline`.
+ */
+export function polylineCenter(latlngs, map) {
+	var i, halfDist, segDist, dist, p1, p2, ratio, center;
+
+	if (!latlngs || latlngs.length === 0) {
+		throw new Error('latlngs not passed');
+	}
+
+	if (!isFlat(latlngs)) {

```

Comment:
I need to check this, in the original function was allowed to pass a multiployline

Reply:
Instead we could document that only first ring is processed for multi-features.